modules/grpc/common/credentials/grpc-credentials-builder.cpp

```diff
@@ -231,6 +231,12 @@ ClientCredentialsBuilder::set_service_account_validity_duration(guint64 validity
   service_account.validity_duration = validity_duration;
 }
 
+void
+ClientCredentialsBuilder::set_application_credentials(const char *key_path)
```

Comment:
Can we call this something similar to `service-account()`'s `key()`? They serve the same purpose.
For example `service-account-key()`?